extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcAuthenticationMechanism.java

```diff
@@ -41,4 +46,15 @@ private boolean isWebApp(RoutingContext context) {
         return OidcTenantConfig.ApplicationType.WEB_APP == tenantContext.oidcConfig.applicationType;
     }
 
+    @Override
+    public Set<Class<? extends AuthenticationRequest>> getCredentialTypes() {
+        return Collections.singleton(TokenAuthenticationRequest.class);
+    }
+
+    @Override
+    public HttpCredentialTransport getCredentialTransport() {
+        //not 100% correct, but enough for now
+        //if OIDC is present we don't really want another bearer mechanism
+        return new HttpCredentialTransport(HttpCredentialTransport.Type.AUTHORIZATION, "bearer");
```

Comment:
That is fine, we can support `bearer,code-flow` in the future when a given mechanism supports different types